python/tracking_benchmark.py

```
@@ -287,55 +297,76 @@ def compute_metrics(self) -> dict[str, Any]:
+                            mc_pdg = self.sim_tree.MCTrack[mc_id].GetPdgCode()
+                            mc_particle = self.PDG.GetParticle(mc_pdg)
+                            if mc_particle:
+                                q_truth = mc_particle.Charge() / 3.0
+                                if q_reco * q_truth > 0:
+                                    n_correct_charge += 1
+                                    h_p_correct_charge.Fill(p_truth)
+                                else:
+                                    n_wrong_charge += 1
+                                    h_p_wrong_charge.Fill(p_truth)
                         except Exception:
                             pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)